boards/posix/native_posix/doc/board.rst

```diff
@@ -389,6 +389,14 @@ The following peripherals are currently provided with this board:
   and provides interrupt priorities. Interrupts can be individually masked or
   unmasked. SW interrupts are also supported.
 
+**Ethernet driver**:
+  A simple TAP based ethernet driver is provided. The driver will create
```

Comment:
One minor comment more from my side: Would it make sense to recommend users to ensure they set NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME when using this driver?

Reply:
NATIVE_POSIX_SLOWDOWN_TO_REAL_TIME is enabled only for tests, so we should be good I guess.